std/format/write.d

```diff
@@ -62,318 +62,6 @@ import std.traits : isSomeString;
    Throws: Mismatched arguments and formats result in a $(D
    FormatException) being thrown.
 
-   Format_String: <a name="format-string">$(I Format strings)</a>
```

Comment:
I suggest to add a link to the new symbol that has for ddoc the old ddoc

Reply:
Sorry, I do not understand, what you want to say with that. What is the "new symbol"? And what does "has for ddoc the old ddoc" mean?

Reply:
a link to the what the new location of the description.